### DISCUSSION TITLE
Kismet

### DISCUSSION THREAD

**Discussion by AndyL316** [WLAN-Pi/feedback#3] [General Inquiries] [answered]
Hi all,

Appologies if this is a stupid question but has Kismet been dropped from WLAN Pi 2.0.x?

thanks
Andy

**Comment by nigelbowden** [comment]
Hi Andy,

Sorry for the delayed response. Yes, it was removed in v2.0.

Nigel.

**Comment by jolla** [comment, marked as answer]
It can be added if desired, but requires compiling from source, instructions can be found here:

https://www.kismetwireless.net/docs/readme/quickstart/
